app.py

```diff
@@ -0,0 +1,97 @@
+import panel as pn
+import hvplot.pandas
+
+# Load Data
+from data import per_dataset_df, all_framework_df
+from widgets import create_selectwidget, create_togglewidget, create_numberwidget
+from plots import create_hvplot,create_table
+
+#clean up framework names
```

Comment:
Looks like many of these commented sections could be moved to an independent function. With those logics live inside functions, we can add unit tests more easily